create-cluster.sh

```diff
@@ -0,0 +1,62 @@
+
+CLUSTER_NAME=example.cluster.k8s # name of the cluster
+STATE_BUCKET=kops-state-bucket # name of s3 bucket where the state of the cluster will live
```

Comment:
STATE_BUCKET name is sensitive - must be removed

Reply:
Ok cool, I added "kops-state-bucket" as a dummy name.